src/Core/Repository/AbstractMultiShopObjectModelRepository.php

```diff
@@ -244,4 +244,111 @@ protected function deleteObjectModelFromShops(ObjectModel $objectModel, array $s
             );
         }
     }
+
+    /**
+     * @param int $id
+     * @param string $objectModelClassName
+     *
```

Comment:
```suggestion
```

Reply:
Yeah actually the PHPDoc is mainly for the return type to specify what's in the array